src/displayapp/screens/ScoreApp.cpp

```diff
@@ -0,0 +1,80 @@
+#include "ScoreApp.h"
+#include "Screen.h"
+#include "Symbols.h"
+#include "lvgl/lvgl.h"
+
+using namespace Pinetime::Applications::Screens;
+
+
+static void btnEventHandler(lv_obj_t* obj, lv_event_t event) {
+  ScoreApp* screen = static_cast<ScoreApp*>(obj->user_data);
+  screen->OnButtonEvent(obj, event);
+}
+
+ScoreApp::widget_t ScoreApp::_createButton(lv_align_t alignment, uint8_t x, uint8_t y, uint8_t width, uint8_t height, const char text[]){
+
+  widget_t button;
+
+  button.button = lv_btn_create(lv_scr_act(), nullptr);
+  button.button->user_data = this;
+  lv_obj_set_event_cb(button.button, btnEventHandler);
+  lv_obj_align(button.button, lv_scr_act(), alignment, x, y);
+  lv_obj_set_width(button.button, width);
+  lv_obj_set_height(button.button, height);
```

Comment:
You can use `lv_obj_set_size(object, width,  height)`

Reply:
Corrected in https://github.com/toitoinou/Pinetime/commit/b5a762a1de2682963eb98bda42a06ee710741935